pkg/server/workspaces/remove.go

```diff
@@ -84,12 +95,6 @@ func (s *WorkspaceService) HandleSuccessfulRemoval(ctx context.Context, workspac
 
 	defer stores.RecoverAndRollback(ctx, s.workspaceStore)
 
-	err = s.revokeApiKey(ctx, fmt.Sprintf("ws-%s", workspaceId))
```

Comment:
I think we can remove the entire `HandleSuccessfulRemoval` function now.
The metadata can also be removed in the remove functions "synchronously".